libraw/src/decode.rs

```diff
@@ -0,0 +1,27 @@
+use crate::Sizes;
+
+use libraw_sys as sys;
+
+pub struct DecodedImage {
```

Comment:
Calling this `RawImage` or something similar might better disambiguate what's actually being offered by this struct.